Crashlytics/Crashlytics/Controllers/FIRCLSContextManager.m

```diff
@@ -0,0 +1,79 @@
+//
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "Crashlytics/Crashlytics/Controllers/FIRCLSContextManager.h"
+
+#import "Crashlytics/Crashlytics/Components/FIRCLSContext.h"
+
+@interface FIRCLSContextManager ()
+
+@property(nonatomic, assign) BOOL hasInitializedContext;
+
+@property(nonatomic, strong) FIRCLSInternalReport *report;
+@property(nonatomic, strong) FIRCLSSettings *settings;
+@property(nonatomic, strong) FIRCLSFileManager *fileManager;
+
+@property(nonatomic, copy) NSString *appQualitySessionId;
+
+@end
+
+@implementation FIRCLSContextManager
+
+- (instancetype)init {
+  self = [super init];
+  if (!self) {
+    return self;
+  }
```

Comment:
Yeah it's a funny pattern - if you check out the style guide there's a part where in init they always do `if (self) { ...set vars }`

go/objc-style#example

This is just a simplified version

Reply:
But the advice about ivars is correct - updating that

Reply:
Thanks for updating to `ivars`.

For the first part, i was referring to returning `self` in a `if` check block. Instead can we do more like below:

```
if (self) {
}
```

instead of 

```
if (!self) {
    return self
}
```